drivers/SmartThings/zigbee-switch/src/multi-switch-no-master/init.lua

```diff
@@ -11,61 +11,114 @@
 -- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 -- See the License for the specific language governing permissions and
 -- limitations under the License.
+local capabilities = require "st.capabilities"
+local st_device = require "st.device"
+local clusters = require "st.zigbee.zcl.clusters"
+local OnOff = clusters.OnOff
 
+local parent_endpoint = 1
 
 local MULTI_SWITCH_NO_MASTER_FINGERPRINTS = {
-  { mfr = "DAWON_DNS", model = "PM-S240-ZB" },
-  { mfr = "DAWON_DNS", model = "PM-S240R-ZB" },
-  { mfr = "DAWON_DNS", model = "PM-S340-ZB" },
-  { mfr = "DAWON_DNS", model = "PM-S340R-ZB" },
-  { mfr = "DAWON_DNS", model = "PM-S250-ZB" },
-  { mfr = "DAWON_DNS", model = "PM-S350-ZB" },
-  { mfr = "DAWON_DNS", model = "ST-S250-ZB" },
-  { mfr = "DAWON_DNS", model = "ST-S350-ZB" },
-  { model = "E220-KR2N0Z0-HA" },
-  { model = "E220-KR3N0Z0-HA" },
-  { model = "E220-KR4N0Z0-HA" },
-  { model = "E220-KR5N0Z0-HA" },
-  { model = "E220-KR6N0Z0-HA" }
+  { mfr = "DAWON_DNS", model = "PM-S240-ZB", children = 1 },
+  { mfr = "DAWON_DNS", model = "PM-S240R-ZB", children = 1 },
+  { mfr = "DAWON_DNS", model = "PM-S250-ZB", children = 1 },
+  { mfr = "DAWON_DNS", model = "PM-S340-ZB", children = 2 },
+  { mfr = "DAWON_DNS", model = "PM-S340R-ZB", children = 2 },
+  { mfr = "DAWON_DNS", model = "PM-S350-ZB", children = 2 },
+  { mfr = "DAWON_DNS", model = "ST-S250-ZB", children = 1 },
+  { mfr = "DAWON_DNS", model = "ST-S350-ZB", children = 2 },
+  { mfr = "ORVIBO", model = "074b3ffba5a045b7afd94c47079dd553", children = 2 },
+  { mfr = "ORVIBO", model = "9f76c9f31b4c4a499e3aca0977ac4494", children = 2 },
```

Comment:
@NikitaKuchta shouldn't this one have 3 child devices ?

Reply:
@KKlimczukS right! I overlooked it, thnx for cathing